dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/WorkflowRepository.java

```diff
@@ -32,47 +32,44 @@
 
 import com.google.common.collect.ImmutableList;
 
-/**
- * cache of process instance id and WorkflowExecuteThread
- */
 @Component
-public class ProcessInstanceExecCacheManagerImpl implements ProcessInstanceExecCacheManager {
+public class WorkflowRepository implements IWorkflowRepository {
```

Comment:
I think we should keep the cache concept because it's not a real dao repository.
```suggestion
public class WorkflowCacheRepository implements IWorkflowRepository {
```

Reply:
Agree.